flocker/common/functional/test_runner.py

```diff
@@ -0,0 +1,93 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Tests for ``flocker.common.runner``.
+"""
+from testtools.matchers import FileContains, DirContains
+
+from twisted.internet import reactor
+
+from flocker.testtools.ssh import (
+    create_ssh_server
+)
+from flocker.common.runner import download, upload
+from flocker.testtools import AsyncTestCase, random_name
+
+
+class UploadTests(AsyncTestCase):
```

Comment:
:question: One could imagine that we should also have download tests, and tests for logging... but... as this is currently just a test util, that is probably overkill.